services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/OdahuServiceImpl.java

```diff
@@ -91,7 +91,9 @@ public OdahuServiceImpl(ProjectService projectService, EndpointService endpointS
     @BudgetLimited
     @Override
     public void create(@Project String project, OdahuCreateDTO odahuCreateDTO, UserInfo user) {
-        Optional<OdahuDTO> odahuDTO = odahuDAO.getByProjectEndpoint(odahuCreateDTO.getProject(), odahuCreateDTO.getEndpoint());
+        Optional<OdahuDTO> odahuDTO = odahuDAO
+                .getByProjectEndpoint(odahuCreateDTO.getProject(), odahuCreateDTO.getEndpoint())
+                .filter(p -> p.getStatus().equals(UserInstanceStatus.FAILED));
```

Comment:
Is it equals to **FAILED** or **not** equals?

Reply:
the Optional filter behavior. If a value is present, and the value matches the given predicate, return an Optional describing the value, otherwise return an empty Optional.